lib/livebook_web/live/session_live.ex

```diff
@@ -1,21 +1,32 @@
 defmodule LivebookWeb.SessionLive do
   use LivebookWeb, :live_view
 
-  alias Livebook.{SessionSupervisor, Session, Delta, Notebook, Runtime}
+  import LivebookWeb.LiveHelpers
+
+  alias Livebook.{SessionSupervisor, Session, Delta, Notebook, Runtime, Users}
 
   @impl true
-  def mount(%{"id" => session_id}, _session, socket) do
+  def mount(%{"id" => session_id}, %{"current_user_id" => current_user_id}, socket) do
```

Comment:
We store `"current_user_id"` in the session, so it again means that running on multiple ports would end up in this key being overriden. However this time scoping the key with port (as with the auth tokens) would mean that we have to do some messy lookup to find the proper `"current_user_id"` in LV. So back to the previous discussions, I think it would be better to be able to specify session `:key` as a function like `key: fn conn -> "#{conn.port}:livebook_key" end`.

Reply:
I don't think we need to worry for this particular case because the id is internal, right? And even if you use different ports, we will be able to reload the information based on the client-side cookie.

Reply:
Yeah, in this case it's actually ok as we have the cookie. But I still think that generally scoping would be convenient.